build.py

```diff
@@ -52,18 +49,22 @@
     )
 
     # There's no reason to use Debug builds of the build_requirements at least
-    # builder.update_build_if(lambda build: True,
-    #                         new_settings={
-    #                             "bison:build_type": "Release",
-    #                             "ruby_installer:build_type": "Release",
-    #                         })
+    builder.update_build_if(
+        lambda build: True,
+        new_settings={
+            "bison:build_type": "Release",
+            "ruby_installer:build_type": "Release",
+            "m4:build_type": "Release",
+        },
+    )
```

Comment:
I had tried to pass a  specific build profile to the docker but failed, cf revert in cd54db977ffb27534343f7434e7823e8e9067540